apps/cookbook/src/app/examples/button-example/examples/_grid-layout.scss

```diff
@@ -0,0 +1,8 @@
+@use '@kirbydesign/core/src/scss/utils';
+
+:host {
+  display: inline-grid;
```

Comment:
Why is the outer display type `inline`? It doesn't look like the layout is used with other inlined elements.

```suggestion
  display: grid;
```

Reply:
As to not spread the buttons thin 😅 
I.e. I would like them to stack up from the left, with the gap in between.
`display: grid` would spread them horizontally across the available width.

Reply:
Yes, I see. As long as no other adjacent elements are displayed inline there's no issue with this 😊